src/types.ts

```diff
@@ -154,7 +155,7 @@ export type OrderBy<
       PaginateBuilder<any, any, any, any, any, any, any, any, any>
     ]
   >
-> = OrderByField<TFields> & OrderByAssociation<TAssociations>
```

Comment:
issue: there's no need for all these indentation changes

Husky should be running ESLint and Prettier before any commit, so I'm not sure why it didn't catch this. I'll add those commands to the CI so at least that will fail if there are any formatting issues. You can run `eslint --fix src` and `prettier --write \"src/**/*.ts\"` to auto-fix any formatting or linting issues. You can also probably install a prettier plugin for whatever editor you're using to format files on save based on a project's `prettierrc`

Reply:
Not sure where those are coming from. My settings don't show a formatter turned on, though it looks like the TypeScript plugin has some formatting options. I'd prefer not to install prettier if I can avoid it.

Reply:
That was just a suggestion. It should be sufficient to run the command.

Reply:
I'll see if I can remember how to run prettier from git.